test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/token/HapiTokenCreate.java

```diff
@@ -241,8 +248,9 @@ protected void updateStateOf(HapiApiSpec spec) {
 			return;
 		}
 		var registry = spec.registry();
-		registry.saveSymbol(token, symbol.orElse(token));
-		registry.saveName(token, name.orElse(token));
+		symbol.ifPresent(s -> registry.saveSymbol(token, s));
+		name.ifPresent(s -> registry.saveName(token, s));
+		registry.saveMemo(token, memo.orElse(""));
```

Comment:
We set memo to empty string because we can show when a memo update happens it changes from an empty string to value set in update transaction ? I forgot why we can't leave this field if not present.